v3/src/components/graph/graph-tests/graph.test.tsx

```diff
@@ -2,7 +2,7 @@ import { render, screen } from "@testing-library/react"
 import React from "react"
 import { DataBroker } from "../../../data-model/data-broker"
 import { DataSet, toCanonical } from "../../../data-model/data-set"
-import {Graph} from "../graph"
+import {Graph} from "../components/graph"
```

Comment:
The convention in most CC projects is to keep the tests in the same folder with the component being tested. There's no compelling reason to prefer one approach over the other. Keeping tests in the same folder means that paths are easier (e.g. the test file can import the tested component from ./component.tsx) and it means that the component and its test are neighbors in code reviews. On the other hand, some people prefer collecting the tests in one place, as you seem to be suggesting here. I'm fine with it either way, but if you do opt for the separate folder I'd name it `tests` rather than `graph-tests`, which is more analogous with the other folders.

Reply:
I'm all for keeping them together. This was an oversight.